projects/govuk-account-manager-prototype/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.7'
+
+x-govuk-account-manager-prototype: &govuk-account-manager-prototype
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: govuk-account-manager-prototype
+  stdin_open: true
+  tty: true
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/govuk-account-manager-prototype
+
+services:
+  govuk-account-manager-prototype-lite:
+    <<: *govuk-account-manager-prototype
+    privileged: true
+    depends_on:
+      - postgres
+      - memcached
+      - govuk-attribute-service-prototype-app
+    environment:
+      MEMCACHE_SERVERS: memcached
+      DATABASE_URL: "postgresql://postgres@postgres/govuk-account-manager-prototype"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/govuk-account-manager-prototype-test"
+      REDIRECT_BASE_URL: http://www.login.service.dev.gov.uk
+      FINDER_FRONTEND_OAUTH_CLIENT_ID: transition-checker-id
+      FINDER_FRONTEND_OAUTH_CLIENT_KEY_UUID: 898d62b7-eed9-464a-a4ae-9d9e08bd9bee
+      FINDER_FRONTEND_OAUTH_CLIENT_KEY: "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIBjlMYhN/1FiZX7RRUyKpqDM2bPIHQswhjm+ZPaOXlgQoAoGCCqGSM49\nAwEHoUQDQgAE2pGXU3YkENblp+SMC8CWpkwPDdWXJdEsXoSQ8dWNYxFFbnXKN4Vv\nRGeLYoNKjuK/2t9rlEEumIzm5EtMxts7Jg==\n-----END EC PRIVATE KEY-----\n"
+
+  govuk-account-manager-prototype-app: &govuk-account-manager-prototype-app
+    <<: *govuk-account-manager-prototype
+    depends_on:
+      - nginx-proxy
+      - postgres
+      - redis
+      - govuk-attribute-service-prototype-app
+      - govuk-account-manager-prototype-worker
+    environment:
+      ASSET_HOST: www.login.service.dev.gov.uk
+      VIRTUAL_HOST: www.login.service.dev.gov.uk
+      BINDING: 0.0.0.0
+      MEMCACHE_SERVERS: memcached
+      DATABASE_URL: "postgresql://postgres@postgres/govuk-account-manager-prototype"
+      ATTRIBUTE_SERVICE_URL: http://attributes.login.service.dev.gov.uk
+      REDIRECT_BASE_URL: http://www.login.service.dev.gov.uk
+      REDIS_URL: redis://redis:6379/0
+      FINDER_FRONTEND_OAUTH_CLIENT_ID: transition-checker-id
+      FINDER_FRONTEND_OAUTH_CLIENT_KEY_UUID: 898d62b7-eed9-464a-a4ae-9d9e08bd9bee
+      FINDER_FRONTEND_OAUTH_CLIENT_KEY: "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEID+UE4jB5QtUFOtekHJliaSB8K2xQU2AS4xczOLAqvCUoAoGCCqGSM49\nAwEHoUQDQgAEjMgE/d6Bdu1K17BTjNycDvIXKSmUgZ5YYoayU3gGqTLhgefuK2qb\no99Wx+SuvdW/GRvIlUIW1ooNRdQ3QFwwCw==\n-----END EC PRIVATE KEY-----\n"
```

Comment:
This too

Reply:
Fixed [here](https://github.com/alphagov/govuk-docker/pull/415/files#diff-35ccf726305b596ebdb1477d3a9a2303R50-R51)